src/components/Filters/AggregationItem/AggregationItem.spec.js

```diff
@@ -0,0 +1,84 @@
+import { render, screen } from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import AggregationItem from './AggregationItem';
+
+const defaultTestProps = {
+  fieldName: 'foo',
+  item: {
+    key: 'hole',
+    value: 'som',
+    doc_count: 100,
+    isDisabled: false,
+  },
+};
+
+const mockStore = configureMockStore();
```

Comment:
Sorry, i forgot to mention that we're trying to move off of mock store and setting up actual redux stores for testing.  

We have utilities to set up redux stores for testing in https://github.com/cfpb/ccdb5-ui/blob/main/src/testUtils/test-utils.js

You can set up a renderComponent function to set up tests.
Take a look at [HasNarrative.spec.js](https://github.com/cfpb/ccdb5-ui/blob/main/src/components/Filters/HasNarrative.spec.js) or [FilterPanelToggle.spec.js](https://github.com/cfpb/ccdb5-ui/blob/main/src/components/Filters/FilterPanelToggle.spec.js) as an example on how we set up tests.

Reply:
This has been updated accordingly